pkg/controllers/cloud/vpc_cache_test.go

```diff
@@ -0,0 +1,97 @@
+package cloud
```

Comment:
missing copy-right

Reply:
Everytime i change the file, copyright goes missing, looks like need to run "make add-copyright" everytime. Didn't know that before.